pkg/reconciler/providerconfig/reconciler_test.go

```diff
@@ -27,6 +27,7 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
@saschagrunert looks like we need to pass in a context at the bottom of the test block here in our call to `Reconcile()`

Reply:
Yes missed that, should be fixed now.